hack/verify-gofmt.sh

```diff
@@ -21,8 +21,8 @@ cd "${OS_ROOT}"
 
 bad_files=$(find_files | xargs gofmt -s -l)
 if [[ -n "${bad_files}" ]]; then
-  echo "!!! gofmt needs to be run on the following files: "
+  >&2 echo "!!! gofmt needs to be run on the following files: "
```

Comment:
the placement of this redirect is confusing to me... why is it at the start of this line, instead of the end of the next? Why do you want this warning message to print to `stderr`?